为什么2.3.11seatunnel 的source StarRocks还需要显示配置schema，但是官方文档显示不需要显示配置

### Search before asking

- [x] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.


### Description

为什么2.3.11seatunnel 的source StarRocks还需要显示配置schema，但是官方文档显示不需要显示配置
Caused by: java.lang.IllegalArgumentException: Schema config can't be null
        at org.apache.seatunnel.api.table.catalog.schema.ReadonlyConfigParser.lambda$parse$0(ReadonlyConfigParser.java:54)
        at java.base/java.util.Optional.orElseThrow(Optional.java:408)
        at org.apache.seatunnel.api.table.catalog.schema.ReadonlyConfigParser.parse(ReadonlyConfigParser.java:53)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.config.StarRocksSourceTableConfig.parseStarRocksSourceConfig(StarRocksSourceTableConfig.java:58)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.config.StarRocksSourceTableConfig.of(StarRocksSourceTableConfig.java:80)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.config.SourceConfig.<init>(SourceConfig.java:54)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.source.StarRocksSourceFactory.createSource(StarRocksSourceFactory.java:75)
        at org.apache.seatunnel.api.table.factory.FactoryUtil.createAndPrepareSource(FactoryUtil.java:179)
        at org.apache.seatunnel.api.table.factory.FactoryUtil.restoreAndPrepareSource(FactoryUtil.java:136)
        ... 8 more


### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/docs/en/connector-v2/source/StarRocks.md
+++ b/docs/en/connector-v2/source/StarRocks.md
@@ -29,7 +29,7 @@ delivers the query plan as a parameter to BE nodes, and then obtains data result
 | database                | string  | yes      | -                 |
 | table                   | string  | no       | -                 |
 | scan_filter             | string  | no       | -                 |
-| schema                  | config  | no       | -                 |
+| schema                  | config  | yes      | -                 |
 | table_list              | array   | no       | -                 |
 | request_tablet_size     | int     | no       | Integer.MAX_VALUE |
 | scan_connect_timeout_ms | int     | no       | 30000             |

--- a/docs/zh/connector-v2/source/StarRocks.md
+++ b/docs/zh/connector-v2/source/StarRocks.md
@@ -21,24 +21,24 @@ import ChangeLog from '../changelog/connector-starrocks.md';
 
 ## 配置选项
 
-| 名称                      | 类型        | 是否必须  | 默认值               |
-|-------------------------|-----------|-------|-------------------|
-| nodeUrls                | list      | 是     | -                 |
-| username                | string    | 是     | -                 |
-| password                | string    | 是     | -                 |
-| database                | string    | 是     | -                 |
-| table                   | string    | 否     | -                 |
-| scan_filter             | string    | 否     | -                 |
-| schema                  | config    | 否     | -                 |
-| table_list              | array     | 否     | -                 |
-| request_tablet_size     | int       | 否     | Integer.MAX_VALUE |
-| scan_connect_timeout_ms | int       | 否     | 30000             |
-| scan_query_timeout_sec  | int       | 否     | 3600              |
-| scan_keep_alive_min     | int       | 否     | 10                |
-| scan_batch_rows         | int       | 否     | 1024              |
-| scan_mem_limit          | long      | 否     | 2147483648        |
-| max_retries             | int       | 否     | 3                 |
-| scan.params.*           | string    | 否     | -                 |
+| 名称                      | 类型        | 是否必须 | 默认值               |
+|-------------------------|-----------|------|-------------------|
+| nodeUrls                | list      | 是    | -                 |
+| username                | string    | 是    | -                 |
+| password                | string    | 是    | -                 |
+| database                | string    | 是    | -                 |
+| table                   | string    | 否    | -                 |
+| scan_filter             | string    | 否    | -                 |
+| schema                  | config    | 是    | -                 |
+| table_list              | array     | 否    | -                 |
+| request_tablet_size     | int       | 否    | Integer.MAX_VALUE |
+| scan_connect_timeout_ms | int       | 否    | 30000             |
+| scan_query_timeout_sec  | int       | 否    | 3600              |
+| scan_keep_alive_min     | int       | 否    | 10                |
+| scan_batch_rows         | int       | 否    | 1024              |
+| scan_mem_limit          | long      | 否    | 2147483648        |
+| max_retries             | int       | 否    | 3                 |
+| scan.params.*           | string    | 否    | -                 |
 
 ### nodeUrls [list]
 

--- a/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/source/StarRocksSourceFactory.java
+++ b/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/source/StarRocksSourceFactory.java
@@ -19,7 +19,7 @@ package org.apache.seatunnel.connectors.seatunnel.starrocks.source;
 
 import org.apache.seatunnel.api.configuration.ReadonlyConfig;
 import org.apache.seatunnel.api.configuration.util.OptionRule;
-import org.apache.seatunnel.api.options.SinkConnectorCommonOptions;
+import org.apache.seatunnel.api.options.ConnectorCommonOptions;
 import org.apache.seatunnel.api.source.SeaTunnelSource;
 import org.apache.seatunnel.api.source.SourceSplit;
 import org.apache.seatunnel.api.table.connector.TableSource;
@@ -50,7 +50,7 @@ public class StarRocksSourceFactory implements TableSourceFactory {
                         StarRocksSourceOptions.PASSWORD,
                         StarRocksSourceOptions.DATABASE)
                 .optional(
-                        SinkConnectorCommonOptions.SCHEMA,
+                        ConnectorCommonOptions.SCHEMA,
                         StarRocksSourceOptions.MAX_RETRIES,
                         StarRocksSourceOptions.QUERY_TABLET_SIZE,
                         StarRocksSourceOptions.SCAN_FILTER,

